lib/rucio/web/rest/webpy/v1/scope.py

```diff
@@ -58,7 +59,7 @@ def GET(self):
         HTTP Error:
             406 Not Acceptable
         """
-        return dumps(list_scopes())
+        return dumps(list_scopes(filter={}, vo=ctx.env.get('vo')))
```

Comment:
As above

Reply:
I also didn't think it needed it, and this bit wasn't in my first (2) commit(s), but I inconsistently got test failures (you can see these in the first two Travis suites, it seemed to happen randomly).
Adding `filter={}` stopped these errors (I think the underlying cause was it converts filter from a string to an `InternalScope`, but in some cases it tried to re-use the last value of filter it had). This was the easiest way I could find to fix this, but I can look back into it if you think it's worth fixing in a different way? Other places in the REST layer (like account) can pass through `{}`, but admittedly only when they don't get given any other parameters.

Reply:
Oh, weird. That's definitely suspicious and could be worth looking into (maybe make a low-priority issue?), but if this fixes it then I'm happy for it to stay in - just wanted to minimise clutter.

Reply:
Created the issue.